src/nvim/map.h

```diff
@@ -8,6 +8,11 @@
 #include "nvim/api/private/dispatch.h"
 #include "nvim/bufhl_defs.h"
 
+#if defined(__NetBSD__)
+#undef uint64_t
+#define uint64_t uint64_t
```

Comment:
Should we instead check `#ifdef uint64_t` in case this scenario is present on systems besides netbsd?

Reply:
As mentioned in some other post, uint64_t is already defined on most systems as identity.
Even those systems will enter the block but do nothing. Anyways, I'll change it if you want to. I checked for NetBSD, so that we could add all NetBSD specific code to that block later on, if at all required.

Reply:
Ok, but at least remove the `#define uint64_t uint64_t` .

Reply:
And please "indent" the inner declaration, by adding a space:

    # undef uint64_t

Reply:
>Ok, but at least remove the #define uint64_t uint64_t .
I did that so that other contributers don't stay clueless on what's happening. Let's say you include a file containing that a definition for uint64_t later on (i.e. after this file has been looked into by the compiler), it will complain that you have multiple definitions.
If you don't define it explicitly in that block, then you'll start getting the same errors as I was getting earlier.